packages/builder/src/pages/builder/updating/index.svelte

```diff
@@ -0,0 +1,31 @@
+<script>
+  import Spinner from "components/common/Spinner.svelte"
+  import { API } from "api"
+
+  setInterval(async () => {
+    const response = await API.get({ url: "/api/migrations/status" })
+    if (!response.migrated) {
+      return
+    }
+
+    const urlParams = new URLSearchParams(window.location.search)
+    const returnUrl = urlParams.get("returnUrl")
+
+    window.location = returnUrl
+  }, 1000)
```

Comment:
Probably want to do this with some kind of back off and jitter to avoid lots of users polling at the same time and DoSing ourselves.